### PR TITLE
Replace mysql-connection-manager with a connection pool

### PR DESCRIPTION
**Pull request by fcarreiro**
Closes issue #50.

### CODE REVIEW

**Comment by fcarreiro** [comment]
@chill117 does this look kind of ok? ;)

File: package.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
     "debug": "2.2.0",
     "depd": "1.1.0",
     "express-session": "1.14.0",
-    "mysql-connection-manager": "0.0.13"
+    "mysql": "^2.11.1"
```
Comment (chill117) [inline package.json]: I prefer hard-coded version numbers for all dependencies.

**Comment by chill117** [comment]
@fcarreiro Other than the one minor issue (dependency version), it looks great. Thanks!

**Comment by fcarreiro** [comment]
Great, thanks for the merge!

**Pull request merged**